raiden-ts/docs-source/transfers.md

```diff
@@ -0,0 +1,61 @@
+# Making Transfers on the dapp
+
+Transfers are off chain payments which raiden nodes make with each other by exchanging messages.
+These are backed by deposits on-chain. When you open a channel you also deposited an amount in the channel which is locked in the token network contract.
+
+We could pay directly to the hub that we opened a channel with or we could pay to anyone else who has a channel with the hub or any node in the chain connected to the hub and indirectly to us via channels.
+
+![raiden-network](https://user-images.githubusercontent.com/15123108/102085833-2d5e1600-3e3d-11eb-917b-eb5311fa118a.png 'Network Visualisation')
+
+Lets walk through how to make a transfer through the dapp
+
+## Direct transfer
+
+If we transfer an amount directly to the hub it is a direct transfer. On the transfer screen we can enter the address and amount and hit continue.
+
+![direct-transfer](https://user-images.githubusercontent.com/15123108/102086318-e9b7dc00-3e3d-11eb-8a1b-cbb4f64034f8.png 'Direct Transfer')
+
+This a direct transfer hence it does not go for route selection. The screen confirms the direct transfer and you could click `Transfer 0.2 TTT` at the bottom to transfer the amount.
```

Comment:
```suggestion
This is a direct transfer because it does not get routed. The screen confirms the direct transfer and you can click `Transfer 0.2 TTT` at the bottom to transfer the amount.
```